app/core/user/user.service.ts

```diff
@@ -132,10 +133,53 @@ export class UserService {
     data += 'FullName=' + user.firstName + ' ' + user.lastName;
     return this.cmsApiService.putUrl('/api/Users/' + user.id, data, {})
       .toPromise()
-      .then(User.extractData)
       .catch(this.handleError);
   }
 
+  getPicture(userId: String): Promise<any> {
+    let headers = new Headers();
+
+    headers.append('authorization', 'Bearer ' + localStorage.getItem('id_token'));
+    headers.append('Access-Control-Allow-Origin', '*');
+  
+    const url = 'http://docker-hip.cs.upb.de:5000/api/Users/' + userId + '/picture';
```

Comment:
No, we never decided to hardcode the URL. We can ALWAYS get at least the url from the config file